packages/fhir-converter/src/templates/cda/Sections/Payers.hbs

```diff
@@ -13,10 +13,11 @@
 
                     Example: https://github.com/HL7/C-CDA-Examples/blob/fb1c33b3a880fc47407f25c5dafa0066996529f4/Documents/CCD/CCD%201/CCD.XML
                     --}}
-                    {{#if (contains (toJsonString this.templateId) '"2.16.840.1.113883.10.20.22.4.87"')}}
+                    {{#if (contains (toJsonString this.templateId) "2.16.840.1.113883.10.20.22.4.87")}}
                         {{#if this.assignedEntity.representedOrganization.name._}}
                             {{>Resources/Organization.hbs org=this.assignedEntity.representedOrganization ID=(generateUUID (toJsonString this.assignedEntity.representedOrganization))}},
-                            {{>References/Coverage/payor.hbs ID=(generateUUID (toJsonString payEntry.act)) REF=(concat 'Organization/' (generateUUID (toJsonString this.assignedEntity.representedOrganization)))}},
+                            {{>References/Coverage/payor.hbs ID=(generateUUID (toJsonString payerEntry.act)) REF=(concat 'Organization/' (generateUUID (toJsonString this.assignedEntity.representedOrganization)))}},
```

Comment:
good well done example of new pattern: 
always generate references to Organizations / Providers etc alongside actually creating the Organization or Provider. This way we dont ever have dangling references!